h2o-docs/src/product/data-munging/downloading-data.rst

```diff
@@ -0,0 +1,72 @@
+Downloading data
+----------------
+
+Sometimes it is desirable to download data from the H2O cluster. For example when computing model predictions, it might be desirable to save these predictions for later.
+
+Download to local memory
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+H2O has functions like ``as_data_frame`` and ``get_frame_data`` in Python and ``as.data.frame`` in R that allow to download the data directly into the client program memory.
```

Comment:
"...that allow to download the data..." -> "...that allow **you** to download the data..."